app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/SelectParticipantsButtonsRow.kt

```diff
@@ -59,7 +62,15 @@ fun SelectParticipantsButtonsAlwaysEnabled(
     onMoreButtonClick: (() -> Unit)? = null,
     onMainButtonClick: () -> Unit,
 ) {
-    SelectParticipantsButtonsRow(count, mainButtonText, true, elevation, modifier, onMoreButtonClick, onMainButtonClick)
+    SelectParticipantsButtonsRow(
+        count = count,
```

Comment:
could we give it some more meaningfull name ? I do not know w hat it counts, maybe selectedPariticpantsCount ?